templates/terraform/examples/route_ilb_beta.tf.erb

```diff
@@ -0,0 +1,52 @@
+resource "google_compute_network" "default" {
+  provider      = "google-beta"
```

Comment:
Can you fix the spacing on this block? Seems to have gotten mixed up

Reply:
ya fixed it.